packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -118,8 +118,10 @@ export class ColorPickerBase extends React.Component<IColorPickerProps, IColorPi
     const props = this.props;
     const strings = this._strings;
     const textLabels = this._textLabels;
-    const { theme, className, styles, alphaSliderHidden } = props;
+    const { theme, className, styles, alphaSliderHidden, useTransparencySlider } = props;
     const { color } = this.state;
+    const alphaValue = color.a;
+    const sliderValue = alphaValue ? this._getAlphaOrTransValue(alphaValue) : 100;
```

Comment:
why do you have this condition, doesn't look correct. If existing alpha is zero - then that is a valid value

Reply:
fixed.